src/workingWithData.ts

```diff
@@ -0,0 +1,54 @@
+import { v4 as uuidv4 } from 'uuid'
+import { resolve } from 'path'
+import { readFile, open, writeFile, unlink, access, mkdir } from 'fs/promises'
+
+type UserInfoType = { name: string; age: number }
+
+export const createFolderIfNotExist = async function (path: string) {
+  const pathToDIr: string = resolve(path)
+  try {
+    await access(pathToDIr)
+  } catch (error) {
+    await mkdir(pathToDIr)
+    console.log('The dir has been created')
+  }
+}
+
+export const createContent = async function (
+  path: string,
+  fileName: string,
+  content: UserInfoType
+) {
+  const takePath: string = resolve(path, fileName)
+
+  try {
+    await access(takePath)
+    await writeFile(takePath, JSON.stringify(content))
+    console.log('Content has been added')
+  } catch (error) {
+    await open(takePath, 'a+')
+    console.log('File was created')
```

Comment:
your final code should not contains a console.log. but you can use custom logger

Reply:
What a custom logger?

Reply:
winston or pino for example